pkg/storaged/storage.css

```diff
@@ -724,3 +664,35 @@ td.storage-action {
 .ct-form-box {
     margin-bottom: 1em;
 }
+
+/* Overview side panels */
+
+#storage .storage-sidebar .table {
+    table-layout: fixed;
+}
+
+.sidepanel-row {
+    display: flex;
+    flex-direction: row;
+}
+
+.sidepanel-row :nth-child(1) {
```

Comment:
Don't use `:nth-child()`; add a class in the HTML (React) and use the class in CSS instead.

Reply:
Done.